lametro/templates/lametro/minutes.html

```diff
@@ -0,0 +1,104 @@
+{% extends "base_with_margins.html" %}
+{% load staticfiles %}
+{% block title %}Minutes{% endblock %}
+
+{% block extra_css %}
+<link rel="stylesheet" type="text/css" href="{% static 'css/jquery-ui.css' %}" />
+<link rel="stylesheet" type="text/css" href="{% static 'css/bootstrap-datetimepicker.min.css' %}" />
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-datepicker/1.4.1/css/bootstrap-datepicker3.css"/>
+{% endblock %}
+
+{% block content %}
+  {% include 'partials/banner.html' %}
+  <div class="row-fluid">
+    <div class="col-sm-9 no-pad-mobile">
+      <br/><br class="non-mobile-only"/>
+      <div id="minutes-form" class="row">
+        <div class="col-xs-8">
+          <form action='/minutes' method='GET'>
+            <div class="input-group" id='date-search'>
+              <span class="input-group-addon" id="sizing-addon3"><i class="fa fa-calendar" aria-hidden="true"></i></span>
+              <input type="text" id="minutes-from" name="minutes-from" class="form-control date-filter" placeholder="Select start date..." value='{{ start_date }}'>
+              <input type="text" id="minutes-to" name="minutes-to" class="form-control date-filter" placeholder="Select end date..." value='{{ end_date }}'>
+              <span class="input-group-btn">
+                <button class="btn btn-default btn-date" id="minutes-search" type="submit"><i class="fa fa-search" aria-hidden="true"></i> <span class='hidden-sm hidden-xs'>Search</span></button>
+              </span>
+            </div>
+          </form>
+        </div>
+        <div class="col-xs-2">
+          <a href="{% url 'lametro:minutes' %}?show=all" class="btn btn-salmon d-inline-block"><i class="fa fa-list" aria-hidden="true"></i><span class="hidden-xs"> All minutes</span></a>
```

Comment:
Hm, clicking this doesn't seem to show all events. Can you take a look? 
<img width="1552" alt="Screen Shot 2022-02-23 at 3 11 57 PM" src="https://user-images.githubusercontent.com/12176173/155410256-56702707-6eaf-43dd-8105-fb7fac0110c9.png">

Reply:
oh shoot, missed this piece, will address & test and let you know when it's ready for your eyes

Reply:
I'm addressing this by removing the query params, but then this link is the same as the reset button's link. I'm not sure whether keeping this interface the same as the events page is more important than avoiding the redundancy of having the same link twice. What do you think @hancush ?

Reply:
You know, that's a great point! Why don't we just remove the All minutes link from the top? We're already loading everything in, and it doesn't make a lot of sense from a UI perspective to have a button at the top of the page expand an element at the bottom of the page, which isn't visible.